web/server/codechecker_server/auth/cc_ldap.py

```diff
@@ -145,14 +145,17 @@ def get_user_dn(con,
     Return the full user dn None if search failed.
     """
 
-    with ldap_error_handler():
-        user_data = con.search_s(account_base_dn, scope, account_pattern)
+    try:
+        with ldap_error_handler():
+            user_data = con.search_s(account_base_dn, scope, account_pattern)
 
-        if user_data:
-            # User found use the user DN from the first result.
-            user_dn = user_data[0][0]
-            LOG.debug("Found user: %s", user_dn)
-            return user_dn
+            if user_data:
+                # User found use the user DN from the first result.
+                user_dn = user_data[0][0]
+                LOG.debug("Found user: %s", user_dn)
+                return user_dn
+    except:
```

Comment:
I still think we shouldn't hide the exception. At least we should print it to the "warning" level log. Same thing in `check_group_membership()`.